### DISCUSSION TITLE
K-WASM 자바 버전 관련해서 질문입니다.

### DISCUSSION THREAD

**Discussion by wonjibkim** [K-PaaS-incubator/k-wasm-app#4] [Q&A]
자바 17로 개발을 진행하시는 이유가 있으실까요?

**Comment by Hyeopgeon-Lee** [comment]
자바 8 버전은 일반적으로 많이 활용되는 버전입니다.
그러나 오라클 공식 홈페이지에 따르면, 자바 8버전은 22년 3월의 끝으로 지원이 종료되었습니다.
따라서 공식 지원되는 버전인 자바 17를 선택하게 되었습니다.

추가적으로 스프링부트 프레임워크 2.7.x 버전의 공식 지원은 2023년 11월 24일 기준으로 종료되었습니다.
이에 스프링부트 프레임워크는 3.x 버전을 공식 출시되었습니다.
3.x 버전은 지원이 종료된 자바8 버전은 더 이상 지원하지 않았습니다.

따라서 본 프로젝트는 스프링부트 프레임워크 3.x 버전을 도입하기로 하였으며, 3.x 버전에 적합한 자바 17를 선택하게 되었습니다.

**Reply by wonjibkim** [reply]
상세한 답변 감사합니다.
